[Streams] Warn about outdated documents during processing

AlejandroFrndz opened the pull request:
Adds an updated_at timestamp to stream definitions and validation so stream records carry a last-modified time.

Updates schema/models and storage mapping to require the field, ensures the server client populates updated_at on upsert, and migrates older records on read by defaulting to the epoch when missing. Updates tests, fixtures, and stories to include the new field.

This enables reliable modification tracking and enforces validation for streams across storage, APIs, and tests.

## Summary

Summarize your PR. If it involves visual changes include a screenshot or gif.


### Checklist

Check the PR satisfies following conditions. 

Reviewers should verify this PR satisfies this list as well.

- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/README.md)
- [ ] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials
- [ ] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
- [ ] If a plugin configuration key changed, check if it needs to be allowlisted in the cloud and added to the [docker list](https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)
- [ ] This was checked for breaking HTTP API changes, and any breaking changes have been approved by the breaking-change committee. The `release_note:breaking` label should be applied in these situations.
- [ ] [Flaky Test Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was used on any tests changed
- [ ] The PR  description includes the appropriate Release Notes section, and the correct `release_note:*` label is applied per the [guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)
- [ ] Review the [backport guidelines](https://docs.google.com/document/d/1VyN5k91e5OVumlc0Gb9RPa3h1ewuPE705nRtioPiTvY/edit?usp=sharing) and apply applicable `backport:*` labels.

### Identify risks

Does this PR introduce any risks? For example, consider risks like hard to test bugs, performance regression, potential of data loss.

Describe the risk, its severity, and mitigation for each identified risk. Invite stakeholders and evaluate how to proceed before merging.

- [ ] [See some risk examples](https://github.com/elastic/kibana/blob/main/RISK_MATRIX.mdx)
- [ ] ...

AlejandroFrndz committed:
feat: Adds updated_at to stream definitions

Adds an updated_at timestamp to stream definitions and validation so stream records carry a last-modified time.

Updates schema/models and storage mapping to require the field, ensures the server client populates updated_at on upsert, and migrates older records on read by defaulting to the epoch when missing. Updates tests, fixtures, and stories to include the new field.

This enables reliable modification tracking and enforces validation for streams across storage, APIs, and tests.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/360151)
* Commit: 3da5b0f2b24c572bb86d5d87cf70bf341d2a37e3

### Failed CI Steps
* [Jest Tests #35](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72ed-7997-4023-bc68-e4d885c24831)
* [Jest Tests #35](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72f3-792c-4bd6-b9ea-e6a6cc17a085)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72ed-7997-4023-bc68-e4d885c24831) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360151/jobs/019a72ed-7997-4023-bc68-e4d885c24831/artifacts/019a72f3-6616-424d-86c6-0c5a99cd601c) Jest Tests #<span></span>35 / content pack export exports selected objects
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72f3-792c-4bd6-b9ea-e6a6cc17a085) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360151/jobs/019a72f3-792c-4bd6-b9ea-e6a6cc17a085/artifacts/019a72f8-f96b-4f71-92c6-fdf0b26142f0) Jest Tests #<span></span>35 / content pack export exports selected objects
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72ed-7997-4023-bc68-e4d885c24831) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360151/jobs/019a72ed-7997-4023-bc68-e4d885c24831/artifacts/019a72f3-6617-4a3e-8425-19deec9b67a6) Jest Tests #<span></span>35 / content pack export renames exported streams
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72f3-792c-4bd6-b9ea-e6a6cc17a085) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360151/jobs/019a72f3-792c-4bd6-b9ea-e6a6cc17a085/artifacts/019a72f8-f96c-4628-b17b-e9cae85ce7ae) Jest Tests #<span></span>35 / content pack export renames exported streams
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72ed-7997-4023-bc68-e4d885c24831) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360151/jobs/019a72ed-7997-4023-bc68-e4d885c24831/artifacts/019a72f3-6614-4892-9ba2-21f086005ab8) Jest Tests #<span></span>35 / content pack import merges imported streams in a target stream
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72f3-792c-4bd6-b9ea-e6a6cc17a085) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360151/jobs/019a72f3-792c-4bd6-b9ea-e6a6cc17a085/artifacts/019a72f8-f96a-46b8-bc43-854f5be5971a) Jest Tests #<span></span>35 / content pack import merges imported streams in a target stream
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72ed-7997-4023-bc68-e4d885c24831) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360151/jobs/019a72ed-7997-4023-bc68-e4d885c24831/artifacts/019a72f3-6615-4177-9942-9937afa8d6dc) Jest Tests #<span></span>35 / migrateOnRead routing status migration should handle definition without routing
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72f3-792c-4bd6-b9ea-e6a6cc17a085) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360151/jobs/019a72f3-792c-4bd6-b9ea-e6a6cc17a085/artifacts/019a72f8-f96a-4b12-8ecd-cce212f4671d) Jest Tests #<span></span>35 / migrateOnRead routing status migration should handle definition without routing
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72ed-7997-4023-bc68-e4d885c24831) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360151/jobs/019a72ed-7997-4023-bc68-e4d885c24831/artifacts/019a72f3-6616-43ad-b792-73acc6ba0f2e) Jest Tests #<span></span>35 / migrateOnRead routing status migration should handle empty routing array
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72f3-792c-4bd6-b9ea-e6a6cc17a085) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360151/jobs/019a72f3-792c-4bd6-b9ea-e6a6cc17a085/artifacts/019a72f8-f96b-4d1a-99c5-06a866f5852c) Jest Tests #<span></span>35 / migrateOnRead routing status migration should handle empty routing array
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72ed-7997-4023-bc68-e4d885c24831) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360151/jobs/019a72ed-7997-4023-bc68-e4d885c24831/artifacts/019a72f3-6618-4d44-a2ef-d6b61bd29aad) Jest Tests #<span></span>35 / migrateOnRead routing status migration when routing rules already have status should preserve existing status field
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360151#019a72f3-792c-4bd6-b9ea-e6a6cc17a085) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360151/jobs/019a72f3-792c-4bd6-b9ea-e6a6cc17a085/artifacts/019a72f8-f96c-4d5f-9e5a-f47512630d6a) Jest Tests #<span></span>35 / migrateOnRead routing status migration when routing rules already have status should preserve existing status field

### History
* :broken_heart: [Build #359893](https://buildkite.com/elastic/kibana-pull-request/builds/359893) failed 84bdf4fe25db4fb6f20264d428862270277bb889
* :broken_heart: [Build #359868](https://buildkite.com/elastic/kibana-pull-request/builds/359868) failed eb97a79319497fa92efe621ece0419552be124c4

cc @AlejandroFrndz

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360151","number":360151,"commit":"3da5b0f2b24c572bb86d5d87cf70bf341d2a37e3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/359893","number":359893,"commit":"84bdf4fe25db4fb6f20264d428862270277bb889"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/359868","number":359868,"commit":"eb97a79319497fa92efe621ece0419552be124c4"}],"number":360151}
buildkite-pr-comment-->

AlejandroFrndz committed:
Set updated_at for wired streams only

Populate updated_at only for wired streams during upsert and bulk upsert operations so only streams that require migration-on-read get stamped. Clear the root stream's embedded timestamp to avoid a runtime-generated value in source, and update tests to use a fixed wired-stream timestamp for deterministic assertions. This prevents unnecessary updates on non-wired streams and makes tests stable.

AlejandroFrndz committed:
Enforces datetime validation for updated_at

Adds stricter validation for the updated_at field by requiring a datetime-formatted string. This prevents invalid timestamp values from passing schema validation and reduces downstream parsing/runtime errors when handling timestamps.

kibanamachine committed:
Changes from node scripts/capture_oas_snapshot   --include-path /api/status   --include-path /api/alerting/rule/   --include-path /api/alerting/rules   --include-path /api/actions   --include-path /api/security/role   --include-path /api/spaces   --include-path /api/streams   --include-path /api/fleet   --include-path /api/saved_objects/_import   --include-path /api/saved_objects/_export   --include-path /api/maintenance_window   --include-path /api/agent_builder --update

kibanamachine committed:
Changes from make api-docs

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/360280)
* Commit: 5850c864b94f6f116ba1375bc42b5a8efb8584dc
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-242425)

### Failed CI Steps
* [Jest Tests #36](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73d9-e57b-4998-b407-33ad8cd39660)
* [Jest Tests #36](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73e1-e583-440e-9dd3-e4a12a5c10c8)
* [FTR Configs #22](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73d9-e5a4-4e14-9d4f-8bdec703f84a)
* [FTR Configs #44](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73d9-e5bb-4519-adec-22c56c29a7bd)
* [FTR Configs #76](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73d9-e5d9-4e7f-ad30-533af6c0a7d3)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73d9-e57b-4998-b407-33ad8cd39660) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360280/jobs/019a73d9-e57b-4998-b407-33ad8cd39660/artifacts/019a73e1-d0e2-4146-bb3e-b6956a5f185d) Jest Tests #<span></span>36 / content pack export exports selected objects
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73e1-e583-440e-9dd3-e4a12a5c10c8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360280/jobs/019a73e1-e583-440e-9dd3-e4a12a5c10c8/artifacts/019a73eb-cf35-4b13-bbb6-c675fdd839d1) Jest Tests #<span></span>36 / content pack export exports selected objects
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73d9-e57b-4998-b407-33ad8cd39660) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360280/jobs/019a73d9-e57b-4998-b407-33ad8cd39660/artifacts/019a73e1-d0e2-43f4-abf6-7e5c1b6e6fe5) Jest Tests #<span></span>36 / content pack export renames exported streams
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73e1-e583-440e-9dd3-e4a12a5c10c8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360280/jobs/019a73e1-e583-440e-9dd3-e4a12a5c10c8/artifacts/019a73eb-cf35-4c7c-9667-603d4f15f878) Jest Tests #<span></span>36 / content pack export renames exported streams
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73d9-e57b-4998-b407-33ad8cd39660) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360280/jobs/019a73d9-e57b-4998-b407-33ad8cd39660/artifacts/019a73e1-d0e0-4f84-b77c-38629dff080e) Jest Tests #<span></span>36 / content pack import merges imported streams in a target stream
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73e1-e583-440e-9dd3-e4a12a5c10c8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360280/jobs/019a73e1-e583-440e-9dd3-e4a12a5c10c8/artifacts/019a73eb-cf34-45e2-9ba0-48d8bc3d00c2) Jest Tests #<span></span>36 / content pack import merges imported streams in a target stream
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73d9-e57b-4998-b407-33ad8cd39660) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360280/jobs/019a73d9-e57b-4998-b407-33ad8cd39660/artifacts/019a73e1-d0df-43d4-8297-6a46d28befd5) Jest Tests #<span></span>36 / migrateOnRead routing status migration should handle definition without routing
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73e1-e583-440e-9dd3-e4a12a5c10c8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360280/jobs/019a73e1-e583-440e-9dd3-e4a12a5c10c8/artifacts/019a73eb-cf34-4d64-b515-cabbf27cc451) Jest Tests #<span></span>36 / migrateOnRead routing status migration should handle definition without routing
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73d9-e57b-4998-b407-33ad8cd39660) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360280/jobs/019a73d9-e57b-4998-b407-33ad8cd39660/artifacts/019a73e1-d0e2-4dee-a28e-b76a039f3f5e) Jest Tests #<span></span>36 / migrateOnRead routing status migration should handle empty routing array
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73e1-e583-440e-9dd3-e4a12a5c10c8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360280/jobs/019a73e1-e583-440e-9dd3-e4a12a5c10c8/artifacts/019a73eb-cf35-4404-ad1a-98ebb15e7585) Jest Tests #<span></span>36 / migrateOnRead routing status migration should handle empty routing array
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73d9-e57b-4998-b407-33ad8cd39660) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360280/jobs/019a73d9-e57b-4998-b407-33ad8cd39660/artifacts/019a73e1-d0e3-4ac2-9763-828970fd2849) Jest Tests #<span></span>36 / migrateOnRead routing status migration when routing rules already have status should preserve existing status field
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360280#019a73e1-e583-440e-9dd3-e4a12a5c10c8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360280/jobs/019a73e1-e583-440e-9dd3-e4a12a5c10c8/artifacts/019a73eb-cf36-49f8-9ebd-0b83c364f315) Jest Tests #<span></span>36 / migrateOnRead routing status migration when routing rules already have status should preserve existing status field

### History
* :broken_heart: [Build #360268](https://buildkite.com/elastic/kibana-pull-request/builds/360268) failed 23c9defefb9511082f5c36b714b5dc6342256595
* :broken_heart: [Build #360250](https://buildkite.com/elastic/kibana-pull-request/builds/360250) failed f779d886d2aeecb6dfc63d900d1ac5d7a8d4afee
* :broken_heart: [Build #360151](https://buildkite.com/elastic/kibana-pull-request/builds/360151) failed 3da5b0f2b24c572bb86d5d87cf70bf341d2a37e3
* :broken_heart: [Build #359893](https://buildkite.com/elastic/kibana-pull-request/builds/359893) failed 84bdf4fe25db4fb6f20264d428862270277bb889
* :broken_heart: [Build #359868](https://buildkite.com/elastic/kibana-pull-request/builds/359868) failed eb97a79319497fa92efe621ece0419552be124c4

cc @AlejandroFrndz

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360280","number":360280,"commit":"5850c864b94f6f116ba1375bc42b5a8efb8584dc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360268","number":360268,"commit":"23c9defefb9511082f5c36b714b5dc6342256595"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360250","number":360250,"commit":"f779d886d2aeecb6dfc63d900d1ac5d7a8d4afee"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360151","number":360151,"commit":"3da5b0f2b24c572bb86d5d87cf70bf341d2a37e3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/359893","number":359893,"commit":"84bdf4fe25db4fb6f20264d428862270277bb889"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/359868","number":359868,"commit":"eb97a79319497fa92efe621ece0419552be124c4"}],"number":360280}
buildkite-pr-comment-->

AlejandroFrndz committed:
Adds updated_at to test fixtures

Includes updated_at fields in mocked wired stream definitions so fixtures match the expected schema and migration/validation logic. Stabilizes tests by replacing a runtime-generated timestamp with a fixed ISO value to avoid timing-related flakiness.

AlejandroFrndz committed:
Adds updated_at migration tests

Adds tests that verify migration adds updated_at to wired streams when missing, preserves an existing updated_at, and does not add updated_at to classic or group streams. Introduces helper factories for classic and group stream definitions and updates test fixtures to validate migration behavior and assertion calls.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/360318)
* Commit: b1712c0b183ac6d590e86e17cb6c170cd57193cb
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-242425)

### Failed CI Steps
* [FTR Configs #2](https://buildkite.com/elastic/kibana-pull-request/builds/360318#019a746d-3199-4baf-9d5c-a2b2a3e0bcc3)
* [FTR Configs #2](https://buildkite.com/elastic/kibana-pull-request/builds/360318#019a7492-8eac-4ecf-a2be-c25095951167)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360318#019a746d-3199-4baf-9d5c-a2b2a3e0bcc3) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360318/jobs/019a746d-3199-4baf-9d5c-a2b2a3e0bcc3/artifacts/019a7492-735b-4221-b71b-d38d1627498e) FTR Configs #<span></span>2 / Embeddable Alerts Table Embeddable alerts panel when user doesnt have access should show a missing authz prompt when the user doesn't have access to a panel's rule types
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360318#019a7492-8eac-4ecf-a2be-c25095951167) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360318/jobs/019a7492-8eac-4ecf-a2be-c25095951167/artifacts/019a749b-b99d-43a2-a5c5-78ff503d3c06) FTR Configs #<span></span>2 / Embeddable Alerts Table Embeddable alerts panel when user doesnt have access should show a missing authz prompt when the user doesn't have access to a panel's rule types

### History
* :broken_heart: [Build #360280](https://buildkite.com/elastic/kibana-pull-request/builds/360280) failed 5850c864b94f6f116ba1375bc42b5a8efb8584dc
* :broken_heart: [Build #360268](https://buildkite.com/elastic/kibana-pull-request/builds/360268) failed 23c9defefb9511082f5c36b714b5dc6342256595
* :broken_heart: [Build #360250](https://buildkite.com/elastic/kibana-pull-request/builds/360250) failed f779d886d2aeecb6dfc63d900d1ac5d7a8d4afee
* :broken_heart: [Build #360151](https://buildkite.com/elastic/kibana-pull-request/builds/360151) failed 3da5b0f2b24c572bb86d5d87cf70bf341d2a37e3
* :broken_heart: [Build #359893](https://buildkite.com/elastic/kibana-pull-request/builds/359893) failed 84bdf4fe25db4fb6f20264d428862270277bb889

cc @AlejandroFrndz

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360318","number":360318,"commit":"b1712c0b183ac6d590e86e17cb6c170cd57193cb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360280","number":360280,"commit":"5850c864b94f6f116ba1375bc42b5a8efb8584dc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360268","number":360268,"commit":"23c9defefb9511082f5c36b714b5dc6342256595"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360250","number":360250,"commit":"f779d886d2aeecb6dfc63d900d1ac5d7a8d4afee"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360151","number":360151,"commit":"3da5b0f2b24c572bb86d5d87cf70bf341d2a37e3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/359893","number":359893,"commit":"84bdf4fe25db4fb6f20264d428862270277bb889"}],"number":360318}
buildkite-pr-comment-->

AlejandroFrndz committed:
docs: Comment updated_at migrate on read

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/360418)
* Commit: b1712c0b183ac6d590e86e17cb6c170cd57193cb
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-242425)

### Failed CI Steps
* [FTR Configs #75](https://buildkite.com/elastic/kibana-pull-request/builds/360418#019a7755-234c-4c79-b019-54bc9e76fa24)
* [FTR Configs #75](https://buildkite.com/elastic/kibana-pull-request/builds/360418#019a777f-1dbf-43df-a362-825e0ff433b3)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360418#019a7755-234c-4c79-b019-54bc9e76fa24) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360418/jobs/019a7755-234c-4c79-b019-54bc9e76fa24/artifacts/019a777e-c2d4-4be6-af35-cc937dabb77b) FTR Configs #<span></span>75 / Embeddable Alerts Table Embeddable alerts panel when user doesnt have access should show a missing authz prompt when the user doesn't have access to a panel's rule types
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/360418#019a777f-1dbf-43df-a362-825e0ff433b3) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/360418/jobs/019a777f-1dbf-43df-a362-825e0ff433b3/artifacts/019a7789-43a1-49bd-8f04-ba58e2b0bcb7) FTR Configs #<span></span>75 / Embeddable Alerts Table Embeddable alerts panel when user doesnt have access should show a missing authz prompt when the user doesn't have access to a panel's rule types

### History
* :broken_heart: [Build #360318](https://buildkite.com/elastic/kibana-pull-request/builds/360318) failed b1712c0b183ac6d590e86e17cb6c170cd57193cb
* :broken_heart: [Build #360280](https://buildkite.com/elastic/kibana-pull-request/builds/360280) failed 5850c864b94f6f116ba1375bc42b5a8efb8584dc
* :broken_heart: [Build #360268](https://buildkite.com/elastic/kibana-pull-request/builds/360268) failed 23c9defefb9511082f5c36b714b5dc6342256595
* :broken_heart: [Build #360250](https://buildkite.com/elastic/kibana-pull-request/builds/360250) failed f779d886d2aeecb6dfc63d900d1ac5d7a8d4afee
* :broken_heart: [Build #360151](https://buildkite.com/elastic/kibana-pull-request/builds/360151) failed 3da5b0f2b24c572bb86d5d87cf70bf341d2a37e3

cc @AlejandroFrndz

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360418","number":360418,"commit":"b1712c0b183ac6d590e86e17cb6c170cd57193cb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360318","number":360318,"commit":"b1712c0b183ac6d590e86e17cb6c170cd57193cb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360280","number":360280,"commit":"5850c864b94f6f116ba1375bc42b5a8efb8584dc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360268","number":360268,"commit":"23c9defefb9511082f5c36b714b5dc6342256595"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360250","number":360250,"commit":"f779d886d2aeecb6dfc63d900d1ac5d7a8d4afee"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/360151","number":360151,"commit":"3da5b0f2b24c572bb86d5d87cf70bf341d2a37e3"}],"number":360418}
buildkite-pr-comment-->
